router/transformer/transformer.go

```diff
@@ -307,13 +305,15 @@ func (trans *HandleT) makeTfProxyRequest(ctx context.Context, proxyReqParams *Pr
 		// This was an error, but not a timeout
 		trans.logger.Errorf(`[TransformerProxy] (Dest-%[1]v) {Job - %[2]v} Client.Do Failure for %[1]v, with %[3]v`, destName, jobId, err.Error())
 		return []byte{}, http.StatusInternalServerError, err
+	} else if resp.StatusCode == http.StatusNotFound {
```

Comment:
os.IsTimeout(err) there is already a function to check if the errors is timeout or not

Reply:
Are you referring to the line where in the error is being casted to net.Error and recommending to change it ? or are you saying the line on which the comment is made needs to be changed ?

Reply:
Also if there is a timeout won't the go HTTP client return `http.StatusGatewayTimeout` , do we need to explicitly send it

Reply:
It seems like the `resp` object will return `nil` and only error which says `context deadline exceeded (Client.Timeout exceeded while awaiting headers)` is being sent from the client

Reply:
> Are you referring to the line where in the error is being casted to net.Error and recommending to change it ? or are you saying the line on which the comment is made needs to be changed ?

Right it is line before it. It works in generic way so better.

Reply:
Done